keps/976-plain-pods/README.md

```diff
@@ -427,58 +406,46 @@ spec:
               cpu: 1m
 ```
 
-#### Groups of Pods with the same shape
+#### Groups of Pods created beforehand
 
-When multiple pods belong to the same group and have the same shape, we need to know how many pods
-belong to the group.
+When a group of pods have different shapes, we need to group them into buckets of similar specs in
+order to create a Workload object.
 
-These groups of Pods can be identified when they have:
+To fully identify the group of pods, the pods need the following:
 - the label `kueue.x-k8s.io/pod-group-name`, as a unique identifier for the group. This should
   be a valid CRD name.
-- the annotation `kueue.x-k8s.io/pod-group-count`, the number of pods to expect in the group.
-  This number is interpreted as int32.
+- The annotation `kueue.x-k8s.io/pod-group-total-count` to indicate how many pods to expect in
+  the group.
+
+The Pod reconciler would group the pods into similar buckets by only looking at the fields that are
+relevant to admission, scheduling and/or autoscaling.
+This list might need to be updated for Kubernetes versions that add new fields relevant to 
+scheduling. The list of fields to keep are:
+- In `metadata`: `labels` (ignoring labels with the `kueue.x-k8s.io/` prefix)
+- In `spec`:
```

Comment:
In the future, we should consider resourceClaim field.

Reply:
Let's add it already, even if Kueue doesn't support it yet.

Reply:
SGTM